sdk/react/ooyalaSkinCore.js

```diff
@@ -153,11 +153,15 @@ OoyalaSkinCore.prototype.handleVideoTouch = function(event) {
  * Hard reset lastPressedTime, either due to button press or otherwise
  */
 OoyalaSkinCore.prototype.handleControlsTouch = function() {
-  Log.verbose("handleControlsTouch - Time reset");
-  this.skin.setState({lastPressedTime: new Date().getTime()});
+  if (this.skin.props.controlBar.autoHide === true) {  
+    Log.verbose("handleVideoTouch - Time set");
```

Comment:
logs are fine - after compiling the jsbundle, logs are removed.  futhermore, events that happen on-click are fine to log out to console, as it won't overflow the console output and gives good insight what happens